frontend/src/helpers/user/api_wrappers.tsx

```diff
@@ -0,0 +1,25 @@
+/* -------------------------------------------------------------------------- */
+/*                      mock backend for user service                     */
+/* -------------------------------------------------------------------------- */
+
+const getProfileUrl = (username: string) => {
+  return "https://i.pravatar.cc/150?u=a042581f4e29026704d";
+};
+
+const getUsername = () => {
+  return "test user";
+};
+
+const getUserPreferences = () => {
```

Comment:
May I ask for this `getUserPReferences`, is this a user input before they start a matching or a preference that the user has prior given and is loaded from the database? If it is the former, I believe you don't really need to have this method in this `api_wrappers`. If you mean the latter, can you elaborate more on how you think about this? Thank you!

Reply:
Yup, I intended for users to be able to save a set of preferences under their Profile so that they do not need to reselect them everytime, and also so that the "Quick Match" button on the Navbar can be used to initiate a matching with these preset preferences. I think I accidentally assumed we would be providing this functionality 😅 What do you think?

Reply:
I see, so we want to allow the user to set their preferences. I am okay with creating such endpoint to support this API call. Will work on the changes in my user service API.

Reply:
> I see, so we want to allow the user to set their preferences. I am okay with creating such endpoint to support this API call. Will work on the changes in my user service API.

Thank you!!